docs/dev/libvirt/README.md

```diff
@@ -151,6 +151,10 @@ so we only need to allow traffic on the libvirt port:
 sudo firewall-cmd --zone=libvirt --add-service=libvirt
 ```
 
+*NOTE:* The installer creates a libvirt network and a `tt0` bridge. In order
+to customize the bridge name, an `install-config.yaml` file is required. See
+[docs/user/overview.md#multiple-invocations](how to create an install-config.yaml file) and [docs/dev/libvirt/customization.md](the libvirt platform customization) documents.
+
```

Comment:
Rest looks good but I don't think there is need to be specific about the bridge name (or any specific option) here (which was my original point). Just say something like "Some aspects of the installation can be customized through the `install-config.yaml file. See..[YOUR LAST SENTENCE ABOVE HERE]".

Reply:
Force pushed a new change. Thanks!